Makefile

```diff
@@ -9,11 +9,11 @@ default:
 	go build -ldflags="-X github.com/navidrome/navidrome/consts.gitSha=$(GIT_SHA) -X github.com/navidrome/navidrome/consts.gitTag=master"
 .PHONY: default
 
-dev: check_env
+dev: check_node_env
```

Comment:
Should we do the same for node version?

Reply:
I think this makes total sense. It crossed my mind then I forgot about it.

> Should we do the same for node version? 

We might as well :)

Reply:
I've tried to keep it as simple and as portable as I could. The only thing I kind of worry about is `sort -V` (version sort). 

From what I could see (like this [StackOverflow thread (2017)](https://unix.stackexchange.com/questions/350315/how-can-i-sort-a-list-with-major-minor-patch-level-and-sometimes-rc-correctly)), the `-V` flag seems supported by default on OpenBSD and FreeBSD, but not on NetBSD.